docs/site_navigation/redirects.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: Redirects
+parent: Site Navigation
+nav_order: 3
+---
+
+# Redirects
+
+The URL redirects are managed in **My Site > Redirects**, both Creators and Developers have access.
+
+## URLs that can't be redirected
+
+Redirects can't be set up **to** or **from** URL paths that start with:
```

Comment:
Isn't this the block list? I.e. can't set up a site page with these urls. 
Particularly /products/* these couldn't be redirected when we joined but this feature has since been added. I believe blog as well now allows for redirects. Did you check with eng?